lib/src/lib.rs

```diff
@@ -165,23 +165,22 @@ impl AccountTable for RawAccountTable {
         let rent_epoch = account_write.rent_epoch as i64;
 
         // TODO: should update for same write_version to work with websocket input
-        let query = postgres_query::query!(
+        let query = query!(
             "INSERT INTO account_write
-            (pubkey_id, slot, write_version, is_selected,
-             owner_id, lamports, executable, rent_epoch, data)
-            VALUES
-            (map_pubkey($pubkey), $slot, $write_version, $is_selected,
-             map_pubkey($owner), $lamports, $executable, $rent_epoch, $data)
-            ON CONFLICT (pubkey_id, slot, write_version) DO NOTHING",
+                    (pubkey, slot, is_selected,
+                    owner, lamports, executable, rent_epoch, data)
+                VALUES
+                ($pubkey, $slot, $is_selected,
+                $owner, $lamports, $executable, $rent_epoch, $data)
+                ON CONFLICT (pubkey_id, slot) DO UPDATE",
             pubkey,
             slot,
-            write_version,
-            is_selected = account_write.is_selected,
+            is_selected,
             owner,
             lamports,
-            executable = account_write.executable,
+            executable,
```

Comment:
this requires account_write.executable/account_write.data/account_write.is_selected as the previous one had.